src/edu/wright/cs/raiderplanner/controller/MenuController.java

```diff
@@ -881,6 +935,21 @@ public void loadModule(Module module, Window previousWindow, ModelEntity previou
 		this.topBox.getChildren().clear();
 		this.title.setText(module.getModuleCode() + " " + module.getName());
 		// =================
+		
+		//Add instructions for the current page
+		HBox instruction = new HBox();
+		GridPane.setHgrow(instruction, Priority.ALWAYS);
+		instruction.setSpacing(50);
+		instruction.setPadding(new Insets(5, 5, 10, 0));
+		this.welcome = new Label(
+				"Welcome " + MainController.getSpc().getPlanner().getUserName() + 
+				"! Use this page to view modules. "
+				+ "Double-click on a module to see more options like "
+					+ "generate Gantish Diagram, or add Tasks/Requirements.");
```

Comment:
I am pretty sure that CheckStyle would dislike the additional indent on this line.